### ISSUE TITLE
NVDA role=alert support for FF and IE

### ISSUE THREAD

**Issue by elenastoyanovaa** [nvaccess/nvda#8797]
Hello,

NVDA does not read a div with a role alert in both FF and IE. It works only in Chrome. Tested with JAWS, and there the issue is not present.

You can test it in the following example: https://jsfiddle.net/elenastoyanovaaa/7pk03rxc/1/
If you remove the aria-label, the issue still persist.

**Comment by ehollig** [comment]
Duplicate of #2442
